app/views/users/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= link_to 'MyProfile', user_path(current_user) %>
+<%= link_to 'Signout', destroy_user_session_path, method: :delete %>
+
+<h1>User profile</h1>
+<strong>Firstname: </strong><%= @user.firstname %> 
```

Comment:
Can we write these kind of lines under <div> or <p> tag and remove <br/>?

Reply:
yes we can! I'll do that